contributing/development/compiling/compiling_for_macos.rst

```diff
@@ -72,20 +68,39 @@ If all goes well, the resulting binary executable will be placed in the
 runs without any dependencies. Executing it will bring up the Project
 Manager.
 
+.. note:: Using a standalone editor executable is not recommended, it should be always packaged into a
```

Comment:
```suggestion
.. note:: Using a standalone editor executable is not recommended, it should be always packaged into an
```
To match the rest (it can be read as either `dot-app` or `app`, the other cases assume the latter)